jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -23,15 +23,150 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
 import java.util.Date;
+import java.util.List;
 
+import static org.eclipse.jkube.kit.common.util.EnvUtil.isWindows;
 import static org.eclipse.jkube.kit.common.util.EnvUtil.loadTimestamp;
 import static org.eclipse.jkube.kit.common.util.EnvUtil.storeTimestamp;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.IsEqual.equalTo;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
 
 public class EnvUtilTest {
 
+    @Test
+    public void testConvertTcpToHttpUrl() {
+        // Given
+        String urlWithHttpsPort = "tcp://0.0.0.0:2376";
+        String urlWithHttpPort="tcp://0.0.0.0:2375";
+        // When
+        String result1 = EnvUtil.convertTcpToHttpUrl(urlWithHttpsPort);
+        String result2 = EnvUtil.convertTcpToHttpUrl(urlWithHttpPort);
+        // Then
+        assertEquals("https://0.0.0.0:2376", result1);
+        assertEquals("http://0.0.0.0:2375", result2);
+    }
+
+    @Test
+    public void testExtractLargerVersion(){
+        //Given
+        String versionA = "4.0.2";
+        String versionB = "3.1.1.0";
+        //When
+        String result2 = EnvUtil.extractLargerVersion(versionA,null);
+        String result3 = EnvUtil.extractLargerVersion(null,versionB);
+        String result4 = EnvUtil.extractLargerVersion(versionA,versionB);
+        String result5 = EnvUtil.extractLargerVersion(versionB,versionA);
+        String result6 = EnvUtil.extractLargerVersion("4.0.0.1","4.0.0");
+        //Then
+        assertNull(EnvUtil.extractLargerVersion(null,null));
+        assertEquals(result2,versionA);
+        assertEquals(result3,versionB);
+        assertEquals(result4,versionA);
+        assertEquals(result5,versionA);
```

Comment:
I think `assertEquals()` expects expected item as the first argument and actual arguments as the second. You need to swap these

Reply:
Understood